pkg/kube/controllers/quarksjob/errand_reconciler.go

```diff
@@ -84,9 +86,30 @@ func (r *ErrandReconciler) Reconcile(request reconcile.Request) (reconcile.Resul
 		return reconcile.Result{}, err
 	}
 
-	if meltdown.NewWindow(r.config.MeltdownDuration, qJob.Status.LastReconcile).Contains(time.Now()) {
-		ctxlog.WithEvent(qJob, "Meltdown").Debugf(ctx, "Resource '%s' is in meltdown, requeue reconcile after %s", request.NamespacedName, r.config.MeltdownRequeueAfter)
-		return reconcile.Result{RequeueAfter: r.config.MeltdownRequeueAfter}, nil
+	negativeTime := metav1.Date(0001, 1, 1, -1, 0, 0, 0, time.UTC)
+	if qJob.Status.LastReconcile.Equal(&negativeTime) || qJob.Status.LastReconcile == nil {
```

Comment:
Why we need to use such `negativeTime` variable? Maybe we can comment this one.

Reply:
Done. Added a comment.